snap/info.go

```diff
@@ -2090,6 +2090,14 @@ func RegistryPlugAttrs(plug *PlugInfo) (account, registry, view string, err erro
 	return account, registry, view, nil
 }
 
+type RefreshFailureSeverity string
+
+const (
+	RefreshFailureSeverityNone        RefreshFailureSeverity = ""
+	RefreshFailureSeverityGeneric     RefreshFailureSeverity = "generic"
+	RefreshFailureSeverityAfterReboot RefreshFailureSeverity = "after-reboot"
```

Comment:
my preference would be for now to have only unset/"" to mean no severity, and "after-reboot*.  "generic" doesn't convey much information